manifest_staging/charts/gatekeeper/templates/gatekeeper-audit-deployment.yaml

```diff
@@ -121,6 +121,8 @@ spec:
         - mountPath: /tmp/audit
           name: tmp-volume
       dnsPolicy: {{ .Values.audit.dnsPolicy }}
+      env:
+        HELMSUBST_DEPLOYMENT_AUDIT_MANAGER_EXTRAENV: ""
```

Comment:
Can you run `make manifests` again to propagate the changes?

Reply:
Done, and pushed.